Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /opt/simulation-framework
+COPY nlisim /opt/simulation-framework/nlisim
```

Comment:
The name of the software is `nlisim`, so this should live at `/opt/nlisim`. I'll push a commit to fix.